gtsfm/utils/io.py

```diff
@@ -250,11 +249,12 @@ def read_cameras_txt(fpath: str) -> Optional[List[Cal3Bundler]]:
             k2 = 0
             calibrations.append(Cal3Bundler(fx, k1, k2, u0, v0))
         elif model == "RADIAL":
-            _, _, img_w, img_h, fx, u0, v0, k1, k2 = cam_params[:9]
-            img_w, img_h, fx, u0, v0, k1, k2 = (
+            _, _, img_w, img_h, fx, fy, u0, v0, k1, k2 = cam_params[:10]
```

Comment:
was this a bug?

Reply:
No we just assumed fx == fy, but NerfStudio requires fx and fy

Reply:
@akshay-krishnan this doesn't correspond to the colmap "Radial" format afaik

https://github.com/colmap/colmap/blob/5c1d58a085920c41cf2d9c892da8041b1fc0c86d/src/colmap/sensor/models.h#L225-L231